Source/Core/DolphinQt2/WiiUpdate.cpp

```diff
@@ -74,46 +119,25 @@ void PerformOnlineUpdate(const std::string& region, QWidget* parent)
               true);
           return !was_cancelled.IsSet();
         },
-        region);
+        std::forward<Args>(args)...);
     Core::QueueHostJob([&dialog] { dialog.close(); }, true);
     return res;
   });
 
   dialog.exec();
+  Core::HostDispatchJobs();
```

Comment:
Yes, we want to be sure all jobs are processed before returning from this function (as we use the dialog, which is going to be destructed after we return from ShowProgress).